### DISCUSSION TITLE
How to collect windows metrics using zabbix agent

### DISCUSSION THREAD

**Discussion by naveenkesari** [apache/skywalking#12397] [Q&A] [answered]
Hi Team,

How to collect windows metrics using zabbix agent, As Linux servers are sending the metrics data to skywalking.

Thanks,
Naveen

**Comment by wu-sheng** [comment, marked as answer]
We don't have UI setup for that. We only release Linux monitoring with OTEL setup.  https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-vm-monitoring/

If you want to use Zabbix, you need to setup MAL and UI by yourself. https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-zabbix/
